interface/github.py

```diff
@@ -45,6 +45,66 @@ def org_has_member(self, username):
         except GithubException as e:
             raise GithubAPIException(e.data)
 
+    def org_get_team(self, id):
+        """Given Github team ID, return team from organization."""
+        try:
+            team = self.org.get_team(id)
+            return team
+        except GithubException as e:
+            raise GithubAPIException(e.data)
+
+    def org_create_team(self, name):
+        """
+        Create team with given name and add to organization.
+
+        :param name: name of team
+        :return: Github team ID
+        """
+        try:
+            team = self.org.\
+                create_team(name, GithubObject.NotSet, "closed", "push")
```

Comment:
What are `closed` and `push` for?

Reply:
Closed is the visibility level of the team - closed means visible to all members
Push is the permission level on the team's repos for everyone in the organization, options are pull/push/admin